File: examples/web/src/main/scala/com/twitter/example/WebLoadTestConfig.scala
package com.twitter.example

import com.twitter.iago.processor.RecordProcessor
import com.twitter.iago.{FinagleParrotConfig, ParrotServerFlags}

class WebLoadTestConfig(config: ParrotServerFlags) extends FinagleParrotConfig(config) {
  override val recordProcessor: RecordProcessor = new WebLoadTest(service)
}

File: examples/web/src/main/java/com/twitter/jexample/WebLoadTestConfig.java
package com.twitter.jexample;

import com.twitter.iago.processor.RecordProcessor;
import com.twitter.iago.FinagleParrotConfig;
import com.twitter.iago.ParrotServerFlags;
import com.twitter.iago.server.FinagleTransport;

public class WebLoadTestConfig extends FinagleParrotConfig {
  public WebLoadTestConfig(ParrotServerFlags config) {
    super(config);
  }

  @Override
  public FinagleTransport transport() {
    return super.transport();
  }

  @Override
  public RecordProcessor recordProcessor() {
    return new WebLoadTest(service());
  }
}
